Jenkins - update election process page for 2022 (#5633)
jenkins-infra/jenkins.io: content/project/board-election-process.adoc

* *October 20* - Voting registration begins
** During the registration period, the election committee will monitor the sign-ups and update the list of voters.
* *November 10* - Nominations deadline
** After the deadline, the election committee will process nominations and notify the candidates
* *November 17* - Voting registration is closed
* *November 17* - Election candidates are published, including personal statements
* *November 17* - Voting begins
** Email is distributed to all registered voters through link:https://civs.cs.cornell.edu/[Condorcet Internet Voting Service]
* *December 02* - Voting ends, 11PM UTC
* *December 07* - Results are announced.

=== Elections Committee

The elections are coordinated by the Jenkins Governance Board members who are not up for re-election this year.
These contributors are responsible for managing the process, preparing the nominee list for elections, forming and verifying the voter list, and processing/announcing the election results.

You can contact the election committee via link:https://community.jenkins.io/g/election-committee[Jenkins Election Committee].
Please use this link for any queries and feedback regarding the elections.

> 

Reviewer: generally recommend to spell out numbers below 10

== Process

> . The Governance Board includes 5 members

Reviewer: ```suggestion
. The Governance Board includes five members
```

** link:/blog/authors/kohsuke[Kohsuke] holds a permanent board seat until such a time he decides to resign.

> ** 4 members are elected.

Reviewer: ```suggestion
** four members are elected.
```

> If link:/blog/authors/kohsuke[Kohsuke] decides to resign, all 5 members will be elected.

Reviewer: ```suggestion
If link:/blog/authors/kohsuke[Kohsuke] decides to resign, all five members will be elected.
```

> . Election occurs every year, normally electing 2 people to the board in each year.

Reviewer: ```suggestion
. Election occurs every year, normally electing two people to the board in each year.
```

> . There will be a period of at least 3 weeks designated for the board to encourage the community to submit nominations.

Reviewer: ```suggestion
. There will be a period of at least three weeks designated for the board to encourage the community to submit nominations.
```

The board will meet in private to select final candidates
. Any company should not dominate in the board (link:/project/board-election-process/#corporate-involvement[see below])

> * The number of board members affiliated with one company must be less than 50%

Reviewer: ```suggestion
* The number of board members affiliated with one company must be less than 50%.
```

In the case of rejection, a justification must be provided to the contributor who submitted the nomination.
The decision may be escalated to the link:/project/governance-meeting/[Jenkins governance meeting] and overridden by voting there.

After receiving and vetting the nominations, the election committee is responsible for contacting the nominees and confirming whether they want to run in the election.
The candidates should also provide their statements, list of affiliations (see <<Corporate Disclosure>>), any account/social media links which may help voters to get more information about them, and plans if they are elected to the role.
This information is published by the election committee before the voting starts.

=== Voter sign-up and eligibility

Any Jenkins project and/or community contributor is eligible to vote in the election if there is a contribution made before September 01 of the election year.
Contributing does not only mean a _code contribution_, link:/participate[contributions] could be:

* Documentation udpates or creation
* Code reviews
* Issue reporting
* Translating resources
* Connecting with or assisting the community
* Code testing

> As long as you are contributing to the Jenkins project or community,  

Reviewer: should there be more after this comma to complete the sentence?

Author: yup, forgot to finish that one, but have updated it now and will submit the update

The election committee is responsible to hold a retrospective for the elections and to make the results of it public.

== Interim Procedures

If a board member resigns, the board is allowed to appoint an interim board member to fulfill the remainder of the term, subject to approval in a regular link:/project/governance-meeting/[governance meeting].

== Corporate Involvement

As an independent community, it is important to us that the Jenkins board does not become overly influenced by any one single corporate entity.

> More on our philosophy can be found in the link:/project/governance[Governance Document].

Reviewer: ```suggestion
You can find more on our philosophy in the link:/project/governance[Governance Document].
```

Minutes link:http://meetings.jenkins-ci.org/jenkins-meeting/2019/jenkins-meeting.2019-09-11-18.04.html[summary] and link:http://meetings.jenkins-ci.org/jenkins-meeting/2019/jenkins-meeting.2019-09-11-18.04.log.html[raw]

* 3 Board positions are elected instead of 2 in the base document (Dean Yu's seat + 2 new seats).
With this change, the 2020 election will have only one board member elected unless a board member steps down.
* Continuous Delivery Foundation will supervise the election.
* We will run the voting using The Condorcet Internet voting system instead of Single Transferable Vote.

Related decisions:

* Introduce a new link:/project/team-leads/#documentation[Documentation officer position] (content officer from the 2015 Proposal).
* All link:/project/team-leads/[officer positions] will be voted on in 2019 and then in 2020.

=== 2015-12-09

Minutes link:http://meetings.jenkins-ci.org/jenkins-meeting/2015/jenkins-meeting.2015-12-09-19.01.html[summary] and link:http://meetings.jenkins-ci.org/jenkins-meeting/2015/jenkins-meeting.2015-12-09-19.01.log.html[raw]

Decisions:

* Formally approve the Governance board election process.
This page represents the process.

> * Expand the board from 3 people to 5 people;

Reviewer: ```suggestion
* Expand the board from three people to five people;
```
